projects/ngneat/transloco/src/lib/transloco.directive.ts

```diff
@@ -101,13 +102,13 @@ export class TranslocoDirective implements OnInit, OnDestroy, OnChanges {
     this.host.nativeElement.innerText = this.translocoService.translate(this.key, this.params, this.langName);
   }
 
-  private structuralStrategy(data: Translation) {
+  private structuralStrategy(data: Translation, limit?: string) {
```

Comment:
since this is only relevant to the structural directive no need to pass is as param, just use it directly.
Also, cache the data result in a variable and use `getValue` to support nesting.
```
private structuralStrategy(data: Translation) {
const translation = this.inlineLimit ? getValue(data, this.inlineLimit) : data;
... 
```